src/Admin/AdherentAdmin.php

```diff
@@ -32,9 +32,9 @@ class AdherentAdmin extends AbstractAdmin
         '_sort_by' => 'registeredAt',
     ];
 
-    public function createQuery($context = 'list')
```

Comment:
`error_reporting | The $context argument of Sonata\AdminBundle\Admin\AbstractAdmin::createQuery is deprecated since 3.3, to be removed in 4.0.`